cmd/config_init.go

```diff
@@ -28,11 +28,12 @@ override the options.`,
 		flags := cmd.Flags()
 		getUserDefaults(flags, &defaults, true)
 		authMethod, auther := getAuthentication(flags)
+		trimmedShellStr := strings.TrimSpace(mustGetString(flags, "shell"))
```

Comment:
I think better would be to initialize correct slice before injecting it into the settings struct. 
```go
var shellCmd []string
trimmedShellStr := strings.TrimSpace(mustGetString(flags, "shell"))
if trimmedShellStr != "" {
	shellCmd = strings.Split(trimmedShellStr, " ")
}
```

Reply:
I agree, updated